src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+// src/redux/store.js
+
+import { createStore, applyMiddleware } from 'redux';
+import { thunk } from 'redux-thunk';
+import reducer from './reducer';
+
+const store = createStore(reducer, applyMiddleware(thunk));
+
+export default store;
```

Comment:
- [ ] [Optional] [Highly Recommended] It would be better if you used `redux-toolkit` instead of regular `redux` to create your store and slice(reducer, actions) because `redux-toolkit` is the new and updated version of redux and you already used it in your react projects before.